librosa/feature/inverse.py

```diff
@@ -195,26 +196,44 @@ def mfcc_to_mel(mfcc, n_mels=128, dct_type=2, norm='ortho', ref=1.0):
     ref : number or callable
         Reference power for (inverse) decibel calculation
 
+    lifter : number >= 0
+        If `lifter>0`, apply *inverse-liftering* (inverse cepstral filtering):
+        `M[n, :] <- M[n, :] / (1 + sin(pi * (n + 1) / lifter)) * lifter / 2`
 
     Returns
     -------
     M : np.ndarray [shape=(n_mels, n)]
         An approximate Mel power spectrum recovered from `mfcc`
 
+    Warns
+    --------
+    RuntimeWarning
+        overflow encountered in power or
+        divide by zero encountered during inverse-liftering.
 
     See Also
     --------
     mfcc
     melspectrogram
     scipy.fftpack.dct
     '''
+    if lifter > 0:
+        mfcc = mfcc.copy()
+        n_mfcc = mfcc.shape[0]
+        mfcc /= 1 + (lifter / 2) \
+            * np.sin(np.pi * np.arange(1, 1 + n_mfcc, dtype=mfcc.dtype) / lifter)[:, np.newaxis]
```

Comment:
declare a different variable `liftered_mfcc`, so that there's no need need for `.copy()`

Reply:
also define the constant `np.sin(np.pi * np.arange(1, 1 + n_mfcc, dtype=mfcc.dtype) / lifter)` on a separate line, with an informative name (`lifter_sine` ?)